app/src/c/main.c

```diff
@@ -90,10 +94,11 @@ static void save_prefs() {
 static void prv_update_app_glance(AppGlanceReloadSession *session, 
                                   size_t limit,
                                   void *context) {
-  APP_LOG(APP_LOG_LEVEL_DEBUG, "prv_update_app_glance");
+  APP_LOG(APP_LOG_LEVEL_DEBUG, "prv_update_app_glance, limit=%d", limit);
+
+  if (limit < 1) return;
```

Comment:
Bad indent

Reply:
This from official pebble example:
https://github.com/pebble-examples/app-glance-hello-world/blob/master/src/app-glance-hello-world.c